web/src/main/java/com/epam/rft/atsy/web/controllers/FileDownloadController.java

```diff
@@ -45,16 +46,16 @@ public Object downloadFile(@PathVariable("candidateId") Long candidateId, Redire
       throws Exception {
 
     CandidateDTO candidateDTO = candidateService.getCandidate(candidateId);
-    String cvFilename = candidateDTO.getCvFilename();
-    if (cvFilename != null && !cvFilename.isEmpty()) {
+    String candidateCVFilename = candidateDTO.getCvFilename();
+    if (candidateCVFilename != null && StringUtils.isNotEmpty(candidateCVFilename)) {
```

Comment:
StringUtils.isNotEmpty method can handle null value, if the parameter is null, then a method will return false value. Check the documentation: https://commons.apache.org/proper/commons-lang/javadocs/api-3.1/org/apache/commons/lang3/StringUtils.html#isEmpty%28java.lang.CharSequence%29